tests/Unit/DTO/ReactionTest.php

```diff
@@ -258,3 +258,41 @@
         ],
     ]);
 });
+
+it('only custom reaction', function () {
```

Comment:
Could you rename this from

`it('only custom reaction)`

To

`test('only custom reaction)`

Please?

Reply:
Ready